src/feature_lib.cpp

```diff
@@ -0,0 +1,29 @@
+#include "feature_lib.h"
+#include "conversions.h"
+#include <opencv2/imgproc.hpp>
```

Comment:
Fix the order according to the code style

Reply:
![image](https://user-images.githubusercontent.com/33483629/180479652-809083bc-4095-4862-8666-1fde4b34acb1.png)
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
That's better?

Reply:
Yep, but remove that blank line